js/everything.js

```diff
@@ -438,8 +453,15 @@ $(function() {
     // How do we know if the value is truly invalid?
     // Preload the form from the URL
     var currentVideoID = getCurrentVideoID();
+    var currentVideoPosition = getCurrentTimePosition();
+    if (currentVideoPosition) {
+        currentVideoPosition = "&t=" + currentVideoPosition;
+    }
+    else {
+        currentVideoPosition = "";
+    }
     if (currentVideoID) {
-        $("#v").attr("value", currentVideoID);
```

Comment:
Does this update the form to be something like `foo&t=123` where `foo` is the video ID? We don't want that. So, we can simplify all of this code